docs/tutorials/how-tos/use-tokens/earn-passive-income-as-a-liquidity-provider.md

```diff
@@ -0,0 +1,67 @@
+---
+html: earn-passive-income-as-a-liquidity-provider.html
+seo:
+    description: Provide liquidity for an Automated Market Maker (AMM) and earn income from trading fees.
+status: not_enabled
+labels:
+  - Decentralized Exchange
+  - Tokens
+  - AMM
+steps: ['Connect', 'Generate', 'Acquire tokens', 'Deposit to the AMM', 'Vote on fees', 'Withdraw from the AMM']
+---
+# Earn Passive Income as a Liquidity Provider
+
+_(Requires the [AMM amendment][])_
+
+Providing liquidity for an [Automated Market Maker (AMM)](../../../concepts/tokens/decentralized-exchange/automated-market-makers.md) can be a great way to earn passive income. By depositing assets into an AMM pool, you'll earn a percentage of the trading fees generated. This tutorial shows how to deposit assets, vote on fees, and withdraw assets.
+
+
+## Prerequisites
+
+- You should be familiar with basic usage of the [xrpl.js client library](https://github.com/XRPLF/xrpl.js/). See [Get Started Using JavaScript](../build-apps/get-started.md) for setup steps.
```

Comment:
This "Get Started Using Javascript" link is broken based on the relative paths of these files. (It'll also have to change again if you move this file.)